### PR TITLE
op-guide: update tidb http api doc

### PR DESCRIPTION
**Pull request by crazycs520**
Update TiDB http api doc.

### CODE REVIEW

**Comment by crazycs520** [comment]
@lilin90 PTAL

File: op-guide/monitor.md [hunk 1]
```diff
@@ -12,8 +12,11 @@ TiDB 集群状态监控目前有两种接口，第一种是通过 HTTP 接口对
 这类接口可以获取组件的一些基本信息，并且可以作为 keepalive 监测接口。另外 PD 的接口可以看到整个 TiKV 集群的详细信息。
 
 ### TiDB Server
+TiDB API 地址：``http://${host}:${port}``。
```
Comment (lilin90) [inline op-guide/monitor.md]:
```suggestion
TiDB API 地址：`http://${host}:${port}`。
```

File: op-guide/monitor.md [hunk 2]
```diff
@@ -12,8 +12,11 @@ TiDB 集群状态监控目前有两种接口，第一种是通过 HTTP 接口对
...
 
-TiDB 对外暴露的 HTTP 接口是 http://host:port/status ，默认的端口号是 10080 （可以通过 --status 参数设置），可以通过访问这个接口获取当前 TiDB Server 的状态，以及判断是否存活。返回结果是 **Json** 格式：
+其中 port 默认为 10080，各类 api_name 详细信息参见 [TiDB API Doc](https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md)。
```
Comment (lilin90) [inline op-guide/monitor.md]:
```suggestion
其中 port 默认为 10080，各类 `api_name` 详细信息参见 [TiDB API Doc](https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md)。
```

File: op-guide/monitor.md [hunk 3]
```diff
@@ -12,8 +12,11 @@ TiDB 集群状态监控目前有两种接口，第一种是通过 HTTP 接口对
...
+
+下面示例是通过访问  ``http://${host}:${port}/status``  获取当前 TiDB Server 的状态，以及判断是否存活。返回结果是 **Json** 格式：
```
Comment (lilin90) [inline op-guide/monitor.md]:
```suggestion
下面示例中，通过访问 `http://${host}:${port}/status` 获取当前 TiDB Server 的状态，以及判断是否存活。返回结果为 **Json** 格式：
```

File: op-guide/monitor.md (hunk 1 above)
Comment (lilin90) [reply op-guide/monitor.md]: Please add a blank line before this line.

File: op-guide/monitor.md [hunk 4]
```diff
@@ -24,12 +28,13 @@ curl http://127.0.0.1:10080/status
 }
 ```
 
-+ connection: 当前 TiDB Server 上的客户端连接数
++ connections: 当前 TiDB Server 上的客户端连接数
```
Comment (lilin90) [inline op-guide/monitor.md]:
```suggestion
+ connections：当前 TiDB Server 上的客户端连接数
```

Comment (lilin90) [reply op-guide/monitor.md]: Use Chinese punctuation in Chinese documents. Please help update it :)

File: op-guide/monitor.md [hunk 5]
```diff
@@ -24,12 +28,13 @@ curl http://127.0.0.1:10080/status
...
 + version: TiDB 版本号
```
Comment (lilin90) [inline op-guide/monitor.md]:
```suggestion
 + version：TiDB 版本号
```

File: op-guide/monitor.md [hunk 6]
```diff
@@ -24,12 +28,13 @@ curl http://127.0.0.1:10080/status
...
 + git_hash: TiDB 当前代码的 Git Hash
```
Comment (lilin90) [inline op-guide/monitor.md]:
```suggestion
 + git_hash：TiDB 当前代码的 Git Hash
```

File: op-guide/monitor.md [hunk 7]
```diff
@@ -24,12 +28,13 @@ curl http://127.0.0.1:10080/status
...
 
 ### PD Server
-PD API 地址： ``http://${host}:${port}/pd/api/v1/${api_name}``。
+
+PD API 地址： `http://${host}:${port}/pd/api/v1/${api_name}`。
```
Comment (lilin90) [inline op-guide/monitor.md]:
```suggestion
PD API 地址：`http://${host}:${port}/pd/api/v1/${api_name}`。
```

**Pull request merged**